Update ecobee.markdown warning (#37080)
home-assistant/home-assistant.github.io: source/_integrations/ecobee.markdown

{% warning %}
As of March 28th, 2024, ecobee is no longer accepting new developer subscriptions, nor are existing developer accounts able to create new API keys. There is no ETA for when they will be allowed again. Existing API keys will continue to function.

> In the meantime, you can use the [HomeKit Device](/integrations/homekit_controller/) integration as a partially functional alternative. 
> 
> When using the HomeKit integration you lose these features:
> - No control over auxiliary heat
> - No list of currently running equipment (can't tell when aux heat is running)
> - No information from air quality sensors
> - No support for vacation features

Reviewer: It is not a task for this integration to describe another integration.

I've adjusted the sentence to not be false anymore as a suggestion.

```suggestion
In the meantime, you can use the [HomeKit Device](/integrations/homekit_controller/) integration as an alternative with limited functionality.
```

Reviewer 2: I guess I disagree since the documentation suggests HomeKit as an alternative. 

I came looking for specific functionality (the ability to control a single hvac system with 2 discreet ecobee devices hooked up) and all this told me was that I should try the HomeKit integration. 

So I wasted several hours finding out that it doesn't do what I want. That could have been saved if I knew specifically which features are absent from the suggested solution. 

I'm not invested enough to make this a fight. I'm not using HA with my ecobees either way. I was just trying to help

Reviewer: > I guess I disagree

That is fine. However, my statement remains.

Reviewer 2: You came back 4 months later to be a dick about this? Quality work, sir!
